operator/src/test/python/test_wdt_mii_filter.py

```diff
@@ -0,0 +1,243 @@
+import os
```

Comment:
Putting comment here... Is __init__.py supposed to be empty?

Reply:
I'm not a python expert but my understanding is that an empty file named "__init__.py" file marks the directory as a python package/module and I added it for python unittest discovery and test execution.  Not currently using any third party tools for unit tests and version of python in our env seems to be 2.7 so using the most basic constructs to run unit tests.

Reply:
Sounds good.